src/com/estafet/learning/US008/Invoice.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+import java.util.Objects;
+
+public abstract class Invoice extends Order{
+    public static final String INVOICE_NAME = "INV";
+    public String OrderNumber;
+    public String ClientDetails;
+    public String ListWithArticles;
+    public String TotalAmount;
+}
+class TradeInvoice extends Invoice {
```

Comment:
there is no point having different classes in one file unless these are not subclasses - please make all classes in separate files

Reply:
changed.